xml/operations-keystone_features.xml

```diff
@@ -58,6 +63,14 @@
    the time a token is issued, the domain-specific roles are not included in
    the token, however, the roles that they map to are.
   </para>
+  <para>
+   <emphasis role="bold">Fernet token provider</emphasis>
+  </para>
+  <para>
+   Provides tokens in the fernet format. This feature is automatically configured
```

Comment:
Should fernet be capitalized? It shows up in the SOC documentation and on the web both capitalized and not. And inconsistently even within the same document.